tests/integration/test_help.sh

```diff
@@ -16,27 +18,46 @@ IFS=$'\n\t'
 
 OUTPUT=$(/bin/true)
 
+# let's clean the environment first
+${ATOMIC} uninstall -f --storage=docker centos || true
+${ATOMIC} uninstall -f --storage=ostree centos || true
```

Comment:
I wasn't able to reproduce either, that's why I put these uninstall commands in the test. Thanks for `atomic images delete`, trying that now.